app/brave_generated_resources.grd

```diff
@@ -389,6 +389,10 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
       <message name="IDS_SETTINGS_WEBRTC_POLICY_DISABLE_NON_PROXIED_UDP" desc="Select value">
         Disable non-proxied UDP
       </message>
+      <!-- Push Messaging -->
+      <message name="IDS_SETTINGS_PUSH_MESSAGING" desc="Select value">
+        Use GCM/FCM for Push Messaging
```

Comment:
@rebron what do you think of this text? Is this good enough for users? or should this be simplified a bit?

Reply:
Change to "Use Google Services for Push Messaging"